monitoring/dashboards/dashboard.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
Seems you still use pkg/errors, how about use pingcap/errors to reduce dependence?

Reply:
thanks, fixed!